README.md

```diff
@@ -128,8 +141,17 @@ All props accept arrays as values for mobile-first responsive styles.
 
 styled-system includes a higher order component to add style props to any component.
 
+**With `styled-components`**
+
+```jsx
+import hoc from 'styled-system/dist/styled-components/hoc'
```

Comment:
Yeah definitely see your point. Although it'd still install deps, what about a separate module in the root for styled-components and glamorous HOCs? e.g.:

```js
import hoc from 'styled-system/styled-components'
// or 
import hoc from 'styled-system/glamorous'
```